profiles/ucentral-ap.yml

```diff
@@ -7,7 +7,7 @@ feeds:
     path: ../../feeds/tip
 
 include:
-  - webui
+  # - webui
```

Comment:
Ah.  I didn't realize we were still doing this.  I thought we had figured out a way to undo the luci configs in minim.yml.  I'd really like it if we only had to make changes in minim.yml and could leave ucentral-ap.yml alone.

Reply:
ok let me recheck by undo the ucentral_ap.yml